common/util.go

```diff
@@ -357,6 +357,10 @@ func IsServiceHandlerRetryableError(err error) bool {
 	return false
 }
 
+func IsNamespaceHandoverError(err error) bool {
+	return err.Error() == ErrNamespaceHandover.Error()
```

Comment:
nit: can you replace other places that perform the same change to use this func? e.g. L341 above.